pkg/patch/jaeger.go

```diff
@@ -0,0 +1,142 @@
+package patch
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"text/template"
+)
+
+const jaegerTemplate = `
```

Comment:
Can you comment in the PR description about the jaeger requirements that result in the solution with the emptyDir, config file, and init container?